rust/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[workspace]
+
+members = [
+    "gitxet",
```

Comment:
nit: For now this is fine, but in the future, we may want to have the gitxet binary be sourced from our public releases when building the plugin.